src/index.js

```diff
@@ -27,14 +27,18 @@
 */
 
 import transform from './transform';
-import createValidator from './validate';
+import validator from './validate';
 import {Transformer} from '@natlibfi/melinda-record-import-commons';
 
 const {startTransformer} = Transformer;
 
 run();
 
 async function run() {
-	const validate = await createValidator();
-	startTransformer(transform, validate);
+	startTransformer(validate);
```

Comment:
`validate` is a bit misleading function name because it actually does the transformation too. Actually, an anonymous function could be used here:
```js
startTransformer(async stream => {
  const records = await transform(stream);
  return validator(records, true, true);
});